data/json/furniture_and_terrain/terrain-fences-gates.json

```
@@ -439,6 +439,60 @@
       "items": [ { "item": "2x4", "count": [ 1, 3 ] }, { "item": "nail", "charges": [ 2, 6 ] }, { "item": "splinter", "count": 1 } ]
     }
   },
+  {
+    "type": "terrain",
+    "id": "t_wattle_fence",
+    "alias": [ "t_wattle_fence_h", "t_wattle_fence_v" ],
+    "name": "woven wattle fence",
+    "description": "A flimsy barrier made of woven sticks. Suitable for small animals.",
```

Two spaces are required between sentences:
```suggestion
    "description": "A flimsy barrier made of woven sticks.  Suitable for small animals.",
```